.github/actions/generate-coverage/tests/test_set_outputs.py

```
@@ -0,0 +1,120 @@
+        job_index="3",
+        runner_os="Ubuntu",
+        runner_arch="X86_64",
+        extra_suffix="Nightly Build",
+    )
+
+    result = module.build_artefact_name(components)
+
+    assert result == "cobertura-coverage-linux-3-ubuntu-x86_64-nightly-build"
+
+
+def test_detect_runner_labels_fallbacks(  # noqa: D103 - docstring via fixture name
```

Ruff (0.14.3)

Unused `noqa` directive (non-enabled: `D103`)

Remove unused `noqa` directive

(RUF100)